### ISSUE TITLE
Base Siren Turn Off via automation

### ISSUE THREAD

**Issue by jxlarrea** [twrecked/hass-aarlo#78]
Hello! I first want to thank you for this component. It made my whole Hassio setup worth it. 

One of my automations trigger the Arlo base siren using `alarm_control_panel.alarm_trigger` when some contact sensors change their state to open. I can't figure out how to turn off the siren via another automation. Is this possible somehow?

**Comment by twrecked** [comment]
Currently it times out - you can configure this in the alarm settings.

I will add a service to stop the siren and a mechanism to quiten it when the alarm is disarmed.

**Comment by jxlarrea** [comment]
I'm pretty new to Home Assistant and Python but I ended up adding a quick and dirty "aarlo_silence" service to the component in "alarm_control_panel.py":

`async def aarlo_siren_service_handler( base,service ):    
    _LOGGER.debug( "{0} siren silenced".format( base.unique_id) )
    base._base.siren_off()`

It works well but it certainly would be better to have an "official" way to do it.

**Comment by twrecked** [comment]
Can you try this branch: https://github.com/twrecked/hass-aarlo/tree/siren-off

I added a `aarl_siren_off` and `aarlo_siren_on` service. I also stop the siren when you disable the alarm.

Call the services like this but adjust the `volume` and `duration` according to taste.:

```
siren_off:
{
"entity_id":"alarm_control_panel.aarlo_front_base",
}

siren_on:
{
"entity_id":"alarm_control_panel.aarlo_front_base",
"volume":"10",
"duration":"30"
}

```

**Comment by jxlarrea** [comment]
@twrecked Hi, tested the new branch, the siren_off service works but it definitely does not turn off the siren when disarmed. 

I noticed that the Arlo base state remains as "Triggered" for as long as the `trigger_time` duration is specified. For example, I manually trigger the alarm, the siren goes on and when I try to disarm it, the Arlo base state wont change at all, it remains as "Triggered" until the siren times out - only then the state changes to `disarmed `and the siren goes silent.

**Comment by twrecked** [comment]
I've just updated the branch with a small piece of debug. I want to make sure the disarm actually gets passed into me.

Can you try again with some extra debug turned on (if you don't all ready have it turned on). Something like this:

```yaml
logger:
    default: info
    logs:
      custom_components.aarlo.alarm_control_panel: debug
```

Thanks.

**Comment by jxlarrea** [comment]
@twrecked Is the branch committed yet? Last update shows up as 4 hours ago.

**Comment by twrecked** [comment]
Commited but not pushed... Sorry. Try now.

**Comment by jxlarrea** [comment]
@twrecked Here are the logs, including the relevant automations:

2019-07-13 11:13:41 INFO (MainThread) [homeassistant.components.automation] Executing [Alarm] Set States when Armed Away
2019-07-13 11:13:41 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Set States when Armed Away: Running script
2019-07-13 11:13:41 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Set States when Armed Away: Executing step call service
2019-07-13 11:13:41 DEBUG (MainThread) [custom_components.aarlo.alarm_control_panel] arlo mode to armed
2019-07-13 11:13:41 DEBUG (EventStream) [custom_components.aarlo.alarm_control_panel] callback:activeMode:armed
2019-07-13 11:13:41 DEBUG (EventStream) [custom_components.aarlo.alarm_control_panel] callback:activeMode:armed
2019-07-13 11:13:45 INFO (MainThread) [homeassistant.components.automation] Executing [Alarm] Trigger while Armed Away
2019-07-13 11:13:45 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Trigger while Armed Away: Running script
2019-07-13 11:13:45 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Trigger while Armed Away: Executing step call service
2019-07-13 11:13:47 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Trigger while Armed Away: Executing step call service
2019-07-13 11:13:47 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Trigger while Armed Away: Executing step call service
2019-07-13 11:13:47 INFO (SyncWorker_7) [custom_components.aarlo.alarm_control_panel] Arlo: triggered
2019-07-13 11:13:47 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Trigger while Armed Away: Executing step call service
2019-07-13 11:13:50 INFO (MainThread) [homeassistant.components.automation] Executing [Alarm] Set States when Alarm Disarmed
2019-07-13 11:13:50 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Set States when Alarm Disarmed: Running script
2019-07-13 11:13:50 INFO (MainThread) [homeassistant.helpers.script] Script [Alarm] Set States when Alarm Disarmed: Executing step call service
2019-07-13 11:13:50 DEBUG (MainThread) [custom_components.aarlo.alarm_control_panel] arlo mode to disarmed
2019-07-13 11:13:50 DEBUG (EventStream) [custom_components.aarlo.alarm_control_panel] callback:activeMode:disarmed
2019-07-13 11:13:50 DEBUG (EventStream) [custom_components.aarlo.alarm_control_panel] callback:activeMode:disarmed
2019-07-13 11:13:52 DEBUG (MainThread) [custom_components.aarlo.alarm_control_panel] arlo siren off

Behavior was the same.

**Comment by twrecked** [comment]
Got it. I pushed a quick fix but I'll work on something neater. The problem is my service call and the standard service call go through two different routes, I'll tie them together.

The `alarm_control_panel.alarm_disarm` call should have worked without the new commit.

**Comment by jxlarrea** [comment]
@twrecked Great! Did you push the fix to the siren_off branch?

**Comment by twrecked** [comment]
Yes. But this is just a quick fix to test the turning off.

**Comment by jxlarrea** [comment]
@twrecked just tested it and works great! Thank you for this. Your component should be the official one!

**Comment by twrecked** [comment]
I've tided the code a little and tested it at my end. I'll merge it back to master later today.

**Comment by twrecked** [comment]
Fixed by #87
